src/session/store/util.ts

```diff
@@ -0,0 +1,86 @@
+import * as keys from "../keys";
+import {IAccessToken, IMap, ISignInInfo, IUserProfile} from "../types";
+
+/**
+ * Generate sign in info from data map.
+ *
+ * This function should take the entire data from the store.
+ */
+export const unmarshalSignInInfo = (data: IMap<any>): ISignInInfo | null => {
+  if (!data) {
+    return null;
+  }
+  const signInInfoData = data[keys.SIGN_IN_INFO];
+  if (!signInInfoData) {
+    return null;
+  }
+
+  let adminPermissions = false;
+  if (keys.ADMIN_PERMISSIONS in signInInfoData) {
+    adminPermissions = signInInfoData[keys.ADMIN_PERMISSIONS] === "1";
+  }
+
+  let signedIn = false;
+  if (keys.SIGNED_IN in signInInfoData) {
+    signedIn = signInInfoData[keys.SIGNED_IN] === 1;
```

Comment:
Wondering why 1 above is a string but here it's a number. Again, what does the 1 signify and could it be defined as a constant that describes the value?

Reply:
Seems that ERIC returns differing data types so safer to work off the actual types that we receive.